# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../aufs2-util/compile.patch.cross
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2009-08-15 16:18:07.000000000 +0200
+++ ./Makefile	2009-08-15 18:54:11.000000000 +0200
@@ -60,7 +60,13 @@
 	}' aufs.in.5 >> $@
 	chmod a-w $@
 
-.INTERMEDIATE: c2sh c2tmac
+.INTERMEDIATE: c2tmac
+c2tmac: c2tmac.c
+	$(HOSTCC) $(CFLAGS) -I$(SYSROOT)/usr/include c2tmac.c -o c2tmac
+
+.INTERMEDIATE: c2sh
+c2sh: c2sh.c
+	$(HOSTCC) $(CFLAGS) -I$(SYSROOT)/usr/include c2sh.c -o c2sh
 
 install_sbin: File = mount.aufs umount.aufs auplink
 install_sbin: Tgt = ${DESTDIR}/sbin
